Jenkinsfile

```diff
@@ -1,169 +1,158 @@
     ])
+  }
+  sh 'rm logcat.txt '
 }
 
 def sendMetrics(String metric, String value) {
-    withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: '5b8ad2d9-61a4-43b5-b4df-b8ff6b1f16fa', passwordVariable: 'influx_pass', usernameVariable: 'influx_user']]) {
-        sh "curl -i -XPOST 'https://greatscott-pinheads-70.c.influxdb.com:8086/write?db=realm' --data-binary '${metric} value=${value}i' --user '${env.influx_user}:${env.influx_pass}'"
-    }
+  withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: '5b8ad2d9-61a4-43b5-b4df-b8ff6b1f16fa', passwordVariable: 'influx_pass', usernameVariable: 'influx_user']]) {
+    sh "curl -i -XPOST 'https://greatscott-pinheads-70.c.influxdb.com:8086/write?db=realm' --data-binary '${metric} value=${value}i' --user '${env.influx_user}:${env.influx_pass}'"
+  }
 }
 
 def sendTaggedMetric(String metric, String value, String tagName, String tagValue) {
-    withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: '5b8ad2d9-61a4-43b5-b4df-b8ff6b1f16fa', passwordVariable: 'influx_pass', usernameVariable: 'influx_user']]) {
-        sh "curl -i -XPOST 'https://greatscott-pinheads-70.c.influxdb.com:8086/write?db=realm' --data-binary '${metric},${tagName}=${tagValue} value=${value}i' --user '${env.influx_user}:${env.influx_pass}'"
-    }
+  withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: '5b8ad2d9-61a4-43b5-b4df-b8ff6b1f16fa', passwordVariable: 'influx_pass', usernameVariable: 'influx_user']]) {
+    sh "curl -i -XPOST 'https://greatscott-pinheads-70.c.influxdb.com:8086/write?db=realm' --data-binary '${metric},${tagName}=${tagValue} value=${value}i' --user '${env.influx_user}:${env.influx_pass}'"
+  }
 }
 
 def storeJunitResults(String path) {
-    step([
-        $class: 'JUnitResultArchiver',
-        testResults: path
-    ])
+  step([
+    $class: 'JUnitResultArchiver',
+    testResults: path
+  ])
 }
 
 def collectAarMetrics() {
-    dir('realm/realm-library/build/outputs/aar') {
-        sh '''set -xe
-        unzip realm-android-library-release.aar -d unzipped
-        find $ANDROID_HOME -name dx | sort -r | head -n 1 > dx
-        $(cat dx) --dex --output=temp.dex unzipped/classes.jar
-        cat temp.dex | head -c 92 | tail -c 4 | hexdump -e '1/4 "%d"' > methods
-        '''
-        sendMetrics('methods', readFile('methods'))
-
-        sendMetrics('aar_size', new File('realm-android-library-release.aar').length())
-
-        def rootFolder = new File('unzipped/jni')
-        rootFolder.traverse (type: DIRECTORIES) { folder ->
-            def abiName = folder.name()
-            def libSize = new File(folder, 'librealm-jni.so').size() as String
-            sendTaggedMetric('abi_size', libSize, 'type', abiName)
-        }
+  dir('realm/realm-library/build/outputs/aar') {
+    sh '''set -xe
+    unzip realm-android-library-release.aar -d unzipped
+    find $ANDROID_HOME -name dx | sort -r | head -n 1 > dx
```

Comment:
Ah right! That's is old code from Christian. I didn't touch that part (apart from the indentation). Agreed, a variable could do, but it's a bit out of the scope of the PR